### PR TITLE
Add store to payment request

### PR DESCRIPTION
**Pull request by daforsman**

### CODE REVIEW

File: src/main/java/com/adyen/model/PaymentRequest.java [hunk 1]
```diff
@@ -44,6 +44,9 @@
     @SerializedName("bankAccount")
     private BankAccount bankAccount = null;
 
+    @SerializedName("store")
+    private Long store = null;
```
[the remark this answers is not in the document]
Comment (daforsman) [reply src/main/java/com/adyen/model/PaymentRequest.java]: Look at that, I changed 👍

**Comment by daforsman** [comment]
Thanks! I dont have merge access, so someone else will have to merge

**Comment by rkewlani** [comment]
> Thanks! I dont have merge access, so someone else will have to merge

Thanks for your contribution. I am merging it.

**Comment by daforsman** [comment]
@rkewlani @lancergr is this only available on test right now? Will a payment request fail if I add this field on prod requests?

**Comment by rkewlani** [comment]
Hi @daforsman,
Your change is available in 1.5.4 release. Are you seeing any issues ?

Thanks,
Rajni

**Pull request merged**